analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnnecessaryUsings2.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System;
```

Comment:
This one is quite far from its sibling. Can you please uniform them more using the namespace prefix?
For example the `ReferencesInsideDocumentationTags` would be good to preserve.